testnet/stacks-node/src/config.rs

```diff
@@ -408,6 +408,33 @@ impl Config {
             burnchain.pox_constants.v1_unlock_height = v1_unlock_height;
         }
 
+        if let Some(epochs) = &self.burnchain.epochs {
+            // Iterate through the epochs vector and find the item where epoch_id == StacksEpochId::Epoch23
+            if let Some(epoch) = epochs
+                .iter()
+                .find(|epoch| epoch.epoch_id == StacksEpochId::Epoch23)
+            {
+                // Override v2_unlock_height to the start_height of epoch2.3
+                debug!(
+                    "Override v2_unlock_height from {} to {}",
+                    burnchain.pox_constants.v2_unlock_height, epoch.start_height
+                );
+                burnchain.pox_constants.v2_unlock_height = epoch.start_height as u32;
```

Comment:
Unclear to me if this needs a `+ 1` to the start_height

Reply:
This should be `Epoch22.start_height + 1` if you want it to most closely match mainnet and testnet (there v2_unlock_height is set to `(BITCOIN_MAINNET_STACKS_22_BURN_HEIGHT as u32) + 1`

Reply:
(Rather than Epoch23 as it is in this code)

Reply:
Thanks -- should be fixed in https://github.com/stacks-network/stacks-blockchain/pull/3705/commits/a13eb368762dc6f0fa1c1f0b102d1870e8884f9c